util/signature/options.go

```diff
@@ -20,6 +20,17 @@ func defaultCfg() *cfg {
 	return new(cfg)
 }
 
+// ErrUnsupportedSignatureScheme is returned when
+// the library does not support a signature scheme
+// that was used for signing a data.
+type ErrUnsupportedSignatureScheme struct {
+	scheme refs.SignatureScheme
```

Comment:
How is it related to this commit?

Reply:
PR adds supporting signature status code and adds some functionality that allows `api-go` callers to understand what exactly error happened

or do you just mean to put that to a separate commit?

please, see the [node PR](https://github.com/nspcc-dev/neofs-node/pull/1532) decs before merging that PR

Reply:
I think we should either create a separate API status or continue using a string message with all wrapping context.
This type just adds another level of abstraction, api-go should be small and reflect API as much as possible.